### ISSUE TITLE
reSTLintBear: Does not find the Pygments package that was installed as a coala-bears dependency

### ISSUE THREAD

**Issue by underyx** [coala-analyzer/coala-bears#526]
I'm getting this after a clean install:
```
README.rst
|  10| ..•code-block::•python
|    | [NORMAL] reSTLintBear:
|    | Cannot analyze code. Pygments package not found.
```

**Comment by sils1297** [comment]
@underyx I can't reproduce it locally, it feels like an upstream problem though. I see however, that our regex shouldn't catch that as an issue and issue a warning instead.

**Comment by sils1297** [comment]
FWIW I do not have pygments installed as it seems and the bear works.

**Comment by underyx** [comment]
Oh, turns out I actually had pygments installed globally already, could you
try to reproduce like that?

**Comment by AbdealiJK** [comment]
:( I can't seem to be able to reproduce this
I can see it happening in a lot of websites with syntax highlighting, and it seems like the reason is because restlint needs docstyle which needs pygments (sometimes?)
But I can't find anyone who has filed a bug about it or solved it !

@underyx Are you still facing it ? Could you try reproducing it ?
